### ISSUE TITLE
embedded wheel selector

### ISSUE THREAD

**Issue by jibaxx** [jasonmamy/cordova-wheel-selector-plugin#11]
It would be great to have this feature without open up a modal. Is it currently possible ? @jasonmamy

**Comment by jasonmamy** [comment]
I think at least in Android it's possible, I'm not sure about iOS.  I
currently can't even build the wheel selector (for Android) in ionic3 as
seems like something in their build is broken, so I can't test it, will
have to wait to see if the bug is fixed (bug seems to be with linking to a
local plugin which worked fine in ionic1 and ionic2).

**Comment by tverilytt** [comment]
Yes would have been great to be able to embed. I would like to show several in same page.
Using Ionic v1..

**Comment by Hanzofm** [comment]
@jasonmamy Some news about this?

**Comment by nkhuynh** [comment]
@jasonmamy Still waiting for you, man! Can we embed wheel selector into any page?

Thanks for your great work.

**Comment by jasonmamy** [comment]
With the current implementation (in Android at least), it uses the
NumberPicker which is used in a Dialog, so without somehow swapping out the
implementation, it seems like it's not easy to make it non-modal.

Thanks
